projects/core/components/root/root.component.ts

```diff
@@ -24,6 +33,15 @@ import {map} from 'rxjs/operators';
     },
 })
 export class TuiRootComponent {
+    @HostBinding('$.class._mobile')
```

Comment:
It will not work this way, because app module does not have NO_ERRORS_SCHEMA on projects. Make all this pipe a private provider, inject elementref and add this property manually (like we do with TUI_MODE) and only use hostlistener + move it to host in Component decorator, otherwise it has to be callable. Again, see how we do it with mode.